mock-core-configs/etc/mock/templates/fedora-eln.tpl

```diff
@@ -38,36 +38,71 @@ cost=2000
 enabled=0
 skip_if_unavailable=False
 
-[eln]
-name=Fedora ELN - Developmental modular packages for the next Enterprise Linux release
+[eln-baseos]
+name=Fedora - ELN BaseOS - Developmental packages for the next Enterprise Linux release
+baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/BaseOS/$basearch/os/
 #metalink=https://mirrors.fedoraproject.org/metalink?repo=eln&arch=$basearch
-baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/Everything/$basearch/os/
 enabled=1
 repo_gpgcheck=0
 type=rpm
 gpgcheck=1
-gpgkey=file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-rawhide-primary
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-eln-$basearch
```

Comment:
We must be sure that this file exists on user machine (be it Fedora, Suse, CentOS). Please add the key to https://github.com/xsuchy/distribution-gpg-keys/ and then use the key from /usr/share/distribution-gpg-keys/* directory.

Reply:
I added the keys to https://github.com/xsuchy/distribution-gpg-keys/tree/master/keys/fedora-eln and I will do a release this evening.

Reply:
@xsuchy Thanks. I'll submit an update to this PR after that release has been done.

Reply:
@xsuchy FYI, the Fedora ELN keys are (and should continue to be) the same as the current Fedora Rawhide keys.

Reply:
@mmathesius Hmm then why the change? What is wrong on `gpgkey=file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-rawhide-primary` then?

Reply:
@xsuchy Two reasons. The first is that `RPM-GPG-KEY-fedora-rawhide-primary` was pointing to the F33 key until your commit a few minutes ago! The second is that I just slurped in the contents of the Fedora ELN repo configuration from https://src.fedoraproject.org/rpms/fedora-repos/blob/master/f/fedora-eln.repo which points to the revised location.

Now that rawhide points to F34, I can revert `gpgkey` to be what you suggested above. Is that what you'd prefer?

Reply:
I just pushed an update setting `gpgkey` to point to the Rawhide primary key. I tested by manually updated the `/usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-rawhide-primary` symlink on my workstation to point to `RPM-GPG-KEY-fedora-34-primary` and all is good.

Reply:
> Is that what you'd prefer?

Yes, please.

Reply:
> > Is that what you'd prefer?
> 
> Yes, please.

@xsuchy Great! That's what I included in https://github.com/rpm-software-management/mock/commit/e1736a46378562adc9ddce8ff84352f791676df0.

Thank you for the feedback. Is there anything else?